src/test/java/step3/RacingCarFirstCollectionTest.java

```diff
@@ -0,0 +1,59 @@
+package step3;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class RacingCarFirstCollectionTest {
+    private final int TEST_POSITIVE_ONE = 1;
+    private final int TEST_NEGATIVE_ONE = -1;
+    private final int RANDOM_NUMBER_LIMIT = 4;
+    private Cars racingCar;
+
+    void init(int carCount, int period) {
+        this.racingCar = Cars.of(carCount, period);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 3, 5})
+    void 입력된_숫자만큼의_객체_생성(int count) {
+        init(count, TEST_POSITIVE_ONE);
+
+        Assertions.assertThat(racingCar.getSize()).isEqualTo(count);
+    }
+
+    @Test
+    void 음수입력시_예외처리() {
+        {
+            assertThatThrownBy(() -> init(TEST_NEGATIVE_ONE, TEST_POSITIVE_ONE)).isInstanceOf(RuntimeException.class).hasMessage("음수는 입력할 수 없습니다.");
```

Comment:
이런식으로 줄바꿈을 해주면 더 읽기 쉬운 코드가 될 것 같아요

```suggestion
            assertThatThrownBy(() -> init(TEST_NEGATIVE_ONE, TEST_POSITIVE_ONE))
                    .isInstanceOf(RuntimeException.class)
                    .hasMessage("음수는 입력할 수 없습니다.");
```

Reply:
감사합니다 🙇  intellij 에 있는 컨벤션만 생각하고 라인에 대해선 신경쓰지 못했네요...